CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
+
+### Release Notes
+* Features:
+  * [Network isolated cluster](https://learn.microsoft.com/azure/aks/concepts-network-isolated) is now Generally Available.
+  * [AKS Security Bulletin](https://learn.microsoft.com/en-us/azure/aks/security-bulletins/overview?tabs=aks-addons%2Caks-cluster%2Caks-node-image) and [AKS CVE Mitigation Status](https://releases.aks.azure.com/webpage/index.html) are now available to track Security and CVE mitigations.
+  * You can now enable [Federal Information Process Standard (FIPS)](https://aka.ms/aks/enable-fips) when using [Arm64 VM SKUs](https://aka.ms/aks/arm64) in Azure Linux 3.0 node pools in Kubernetes version 1.31+.
+  * Custom Certificate Authority is available as GA in the [2025-01-01 GA API](https://learn.microsoft.com/rest/api/aks/managed-clusters/create-or-update?view=rest-aks-2025-01-01&tabs=HTTP#create-managed-cluster-with-custom-ca-trust-certificates). It isn't yet available in the CLI until May 2025. To use the GA feature in CLI before release, you can use the [`az rest`](https://learn.microsoft.com/cli/azure/reference-index?view=azure-cli-latest#az-rest) command to add custom certificates during cluster creation. For more information, see [aka.ms/aks/custom-certificate-authority](https://aka.ms/aks/custom-certificate-authority).
+
+* Bug Fixes:
+  * Fix an [issue](https://github.com/azure-networking/cilium-private/pull/465)) in [Azure CNI Powered by Cilium](https://learn.microsoft.com/azure/aks/azure-cni-powered-by-cilium), if the DNS request timeout is set to below 2 seconds, the high-scale scenarios may cause drops due to duplicate request ID.
+  * Fix an issue where [load balancer](https://learn.microsoft.com/azure/aks/load-balancer-standard) tags were not updated after a managed cluster tag update. Load balancer tags now correctly reflect the latest state.
+  * Fix an [issue](https://github.com/cilium/cilium/issues/38210) in Cilium v1.17 where a deadlock causing pods unable to start. 
+
+* Behavior Changes:
+  * `aksmanagedap` is blocked as a reserved name for use of creating agent pool.
+  * `linuxutil plugin` is temporarily disabled for [Retina Basic and ACNS](https://retina.sh/docs/Metrics/plugins/Linux/linuxutil) for it is causing memory leaks that leads to Retina pods OOMKill.
+  * ACNS configmaps (`cilium`, `retina`, `hubble`) now auto‑format cluster names to satisfy Cilium 1.17 rules:≤ 32 chars, lowercase alphanumeric characters and dashes, no leading/trailing dashes. See this [link](https://github.com/cilium/cilium/commit/b1f10bcb26d71f046d8ad108d5f4b9e42aa5d486) for details.
+  * The `defaultConfig.gatewayTopology` field is now included in the Istio add-on `MeshConfig` AllowList as an unsupported field. For more details, see the [Istio MeshConfig documentation](https://learn.microsoft.com/azure/aks/istio-meshconfig#allowed-supported-and-blocked-meshconfig-values).
```

Comment:
I don't think this is a behavior change as there are no existing users for this and thus wouldn't be seeing a behavior change. IMO this should be under features or preview features section. @biefy can you confirm if this is in preview/stable and we can accordingly move it to the relevant section?

Reply:
Keep this open for now

Reply:
We used to disallow this config.  Now it is allowed now.  It is a config come with native Istio and we are not doing any customization.  We just start allowing updating it.  From disallowing to allowing should fit in behavior change.

Reply:
On 2nd though, @charleswool , I think it better fit into components update section.   It is more of a component (Istio) update rather than a AKS behavior change.